python/ffsim/qiskit/sim.py

```diff
@@ -767,6 +801,44 @@ def _extract_x_gates(circuit: QuantumCircuit) -> tuple[list[int], QuantumCircuit
     return indices, remaining_circuit
 
 
+def _apply_permutation_gate_vec(
```

Comment:
The name `_apply_permutation_gate` would be more consistent with the other function names in this file.